text-editor/text-editor-003-en.md

```diff
@@ -1 +1,102 @@
 ## Insertion of a tweet : a case study
+
+In this chapter, we will try to give a better insight of how the whole thing is
+working, by breaking down the insertion of a tweet case and the components
+involved in it's workflow.
+
+### Getting the HTML markup from twitter
+
+Keep in mind that our main goal here is to obtain the simplest possible HTML
+representation of our tweet, so it can lately be saved to the API. That way it
+will be easy for us to parse it as a component, and finally display it to the
+end user in the front end application.
+
+The following chart will give you an overview of the actions flow :
+![Insert tweet action flow chart](https://github.com/jaljo/articles/raw/314724209eb27b7b1d61795341934c4783084bee/images/text-editor/RTE.png)
```

Comment:
You can directly set the path to `/images/text-editor/RTE.png` so it'll always display the version of this file corresponding to the version of the `.md` file (same revision)

Reply:
Nice trick, I was not aware of that :+1: